### DISCUSSION TITLE
ScriptReference/Editor.OnInspectorGUI

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#252] [Notes]
# ScriptReference/Editor.OnInspectorGUI

https://docs.unity3d.com/ScriptReference/Editor.OnInspectorGUI.html

**Comment by radiatoryang** [comment]
There's several ways to get the "width" of the Inspector window, see [here](https://forum.unity.com/threads/editorguilayout-get-width-of-inspector-window-area.82068/).  
- [`EditorGUIUtility.currentViewWidth`](https://docs.unity3d.com/ScriptReference/EditorGUIUtility-currentViewWidth.html) might help.  
- [`EditorGUIUtility.fieldWidth`](https://docs.unity3d.com/ScriptReference/EditorGUIUtility-fieldWidth.html) is supposed to take the Inspector scrollbar into account.  
There's also a *nasty* rumor that Screen.width will return the width of the inspector, when called from OnInspectorGUI() ?!?!...
